src/devices/src/virtio/balloon/device.rs

```diff
@@ -41,11 +41,21 @@ macro_rules! mem_of_active_device {
     };
 }
 
+fn mb_to_pages(amount_mb: u32) -> Result<u32, BalloonError> {
+    amount_mb
+        .checked_mul(MB_TO_4K_PAGES)
+        .ok_or(BalloonError::TooManyPagesRequested)
+}
+
+fn pages_to_mb(amount_pages: u32) -> u32 {
+    amount_pages / MB_TO_4K_PAGES
+}
```

Comment:
Do you treat cases where # of pages is not multiple of MBs? Should there be a comment here at least explaining why that shouldn't happen or smth?

200 pages here returns 0 MB for example.